roda-core/roda-core/src/main/resources/config/roda-core.properties

```diff
@@ -529,4 +529,10 @@ core.synchronization.scheduleInfo=0 0 * * *
 #
 # Status: in use
 ##########################################################################
-core.aip.lockToEdit=false
\ No newline at end of file
+core.aip.lockToEdit=false
+
+##########################################################################
+# marketplace api url
+#
+##########################################################################
+core.market.info.url=http://api.market.roda.staging.labs.keep.pt/api/v2/marketplace
```

Comment:
This should not be set by default in the code, let each implementation decide the URL it will consume.

Reply:
done